runWebpack.sh

```diff
@@ -160,6 +160,7 @@ if [[ "$NODE_ENV" == "dev" || "$NODE_ENV" == "auto" ]]; then
   echo ./node_modules/.bin/webpack-dev-server
   ./node_modules/.bin/webpack-dev-server
 else
+  export NODE_OPTIONS='--max-old-space-size=4096'
```

Comment:
what is this?

Reply:
this is to allow node to allowcate more mem (default is 512MB i think). 

we have this in the Dockerfile but don't know why not here. And portal is so fat that it need this to spin up...

Reply:
ok, is this related to covid19? if not, can we make a separate PR to merge this to master? this PR will only be merged to branch feat/covid19dashboard for now

Reply:
yeah portal on this branch won't start up if don't have this 😞 

i guess by adding more into portal eventually the master branch will gonna need this one day

Reply:
just adding 1 environment variable triggered this?? 😱

Reply:
it sounds unrealistic but 😕